draft-ietf-oauth-identity-chaining.md

```diff
@@ -293,6 +284,19 @@ Authorization Servers SHOULD follow the OAuth 2.0 Security Best Current Practice
 
 --- back
 
+# Use cases
+
+This sections outlines some use cases the authors have identified where identity chaining as described in this document should be used. This section is not complete and other use cases not mentioned here are also valid.
```

Comment:
Suggest a minor change in wording here (that also anticipates the likely merge of #76):

```suggestion
This sections outlines some use cases where the identity and authorization chaining described in this document can be applied. This section is not complete and other use cases not mentioned here are also valid.
```